packages/mobile/src/components/lineup/Lineup.tsx

```diff
@@ -514,9 +518,10 @@ export const Lineup = ({
         onScroll={handleScroll}
         ListHeaderComponent={header}
         ListFooterComponent={<View style={{ height: 16 }} />}
+        ListEmptyComponent={LineupEmptyComponent}
         onEndReached={handleLoadMore}
         onEndReachedThreshold={LOAD_MORE_THRESHOLD}
-        sections={sections}
+        sections={areSectionsEmpty(sections) ? [] : sections}
```

Comment:
Interesting, sg

Reply:
Yeah this was somewhat annoying, but section list doesn't handle this by ddefault L:;(

Reply:
If it has sections, even if their all empty, it thinks there's stuff

Reply:
Ahh okay that makes a lot of sense actually consider it doesn't handle that by default

Reply:
Can we add that to section list then?

Reply:
Oh yeah I forgot we had our own. sure thing bossman

Reply:
DONEZO